src/main/resources/assets/admin/common/js/form/FormItemOccurrences.ts

```diff
@@ -294,7 +294,7 @@ export class FormItemOccurrences<V extends FormItemOccurrenceView> {
             return Q(null);
         }
 
-        const occurrenceView: V = this.createNewOccurrenceView(occurrence);
+        const occurrenceView: V = this.createNewOccurrenceView(occurrence, isNew);
```

Comment:
Maybe `createNewOccurrenceView` should be renamed to `createOccurrenceView` (without "New"?)